server/api-manager.js

```diff
@@ -1020,6 +1022,30 @@ class MindbodyQueries {
     }
     return false;
   }
+
+  loadConfig(filename) {
+    fs.readFile(filename, "utf8", (err, jsonString) => {
+      if (err) {
+        console.log("File read failed:", err);
+        return;
+      }
+      try {
+        const data = JSON.parse(jsonString);
+        if (filename === "./server/config/secrets.json")
+        {
```

Comment:
The bracket placement is inconsistent here. It should also load the username and password, because then there would be fewer pull requests.

Reply:
The code in this pr does load the username and password

Reply:
Yeah, Jason, please do change the bracket spacing.
I can confirm that username and password are loaded in the full files. I don't see them in git, but that's just an error in how its displayed here, I think.